INFO:    Using cached SIF image
INFO:    Using cached SIF image
INFO:    Using cached SIF image
INFO:    Using cached SIF image
INFO:    Using cached SIF image
INFO:    Using cached SIF image
INFO:    Using cached SIF image
INFO:    Using cached SIF image
INFO:    gocryptfs not found, will not be able to use gocryptfs
INFO:    gocryptfs not found, will not be able to use gocryptfs
INFO:    gocryptfs not found, will not be able to use gocryptfs
INFO:    gocryptfs not found, will not be able to use gocryptfs
INFO:    gocryptfs not found, will not be able to use gocryptfs
INFO:    gocryptfs not found, will not be able to use gocryptfs
INFO:    gocryptfs not found, will not be able to use gocryptfs
INFO:    gocryptfs not found, will not be able to use gocryptfs
Traceback (most recent call last):
  File "/work2/10756/piercezhang9871/stampede3/3d-1d/study_calibrate.py", line 38, in <module>
Traceback (most recent call last):
  File "/work2/10756/piercezhang9871/stampede3/3d-1d/study_calibrate.py", line 38, in <module>
    MESH_CACHE_DIR = os.environ.get("MESH_CACHE_DIR", os.path.join(os.environ.get("SCRATCH", "/tmp"), "3d-1d", ".mesh_cache"))
                     ^^
NameError: name 'os' is not defined. Did you forget to import 'os'?
Traceback (most recent call last):
  File "/work2/10756/piercezhang9871/stampede3/3d-1d/study_calibrate.py", line 38, in <module>
    MESH_CACHE_DIR = os.environ.get("MESH_CACHE_DIR", os.path.join(os.environ.get("SCRATCH", "/tmp"), "3d-1d", ".mesh_cache"))
                     ^^
NameError: name 'os' is not defined. Did you forget to import 'os'?
Traceback (most recent call last):
  File "/work2/10756/piercezhang9871/stampede3/3d-1d/study_calibrate.py", line 38, in <module>
    MESH_CACHE_DIR = os.environ.get("MESH_CACHE_DIR", os.path.join(os.environ.get("SCRATCH", "/tmp"), "3d-1d", ".mesh_cache"))
                     ^^
NameError: name 'os' is not defined. Did you forget to import 'os'?
Traceback (most recent call last):
  File "/work2/10756/piercezhang9871/stampede3/3d-1d/study_calibrate.py", line 38, in <module>
    MESH_CACHE_DIR = os.environ.get("MESH_CACHE_DIR", os.path.join(os.environ.get("SCRATCH", "/tmp"), "3d-1d", ".mesh_cache"))
                     ^^
NameError: name 'os' is not defined. Did you forget to import 'os'?
Traceback (most recent call last):
  File "/work2/10756/piercezhang9871/stampede3/3d-1d/study_calibrate.py", line 38, in <module>
    MESH_CACHE_DIR = os.environ.get("MESH_CACHE_DIR", os.path.join(os.environ.get("SCRATCH", "/tmp"), "3d-1d", ".mesh_cache"))
                     ^^
NameError: name 'os' is not defined. Did you forget to import 'os'?
Traceback (most recent call last):
  File "/work2/10756/piercezhang9871/stampede3/3d-1d/study_calibrate.py", line 38, in <module>
    MESH_CACHE_DIR = os.environ.get("MESH_CACHE_DIR", os.path.join(os.environ.get("SCRATCH", "/tmp"), "3d-1d", ".mesh_cache"))
                     ^^
NameError: name 'os' is not defined. Did you forget to import 'os'?
Traceback (most recent call last):
  File "/work2/10756/piercezhang9871/stampede3/3d-1d/study_calibrate.py", line 38, in <module>
    MESH_CACHE_DIR = os.environ.get("MESH_CACHE_DIR", os.path.join(os.environ.get("SCRATCH", "/tmp"), "3d-1d", ".mesh_cache"))
                     ^^
NameError: name 'os' is not defined. Did you forget to import 'os'?
    MESH_CACHE_DIR = os.environ.get("MESH_CACHE_DIR", os.path.join(os.environ.get("SCRATCH", "/tmp"), "3d-1d", ".mesh_cache"))
                     ^^
NameError: name 'os' is not defined. Did you forget to import 'os'?
srun: error: c454-072: tasks 0-7: Exited with exit code 1
